routes/add_to_slack.html

```diff
@@ -0,0 +1 @@
+<a href="https://slack.com/oauth/authorize?scope=incoming-webhook&client_id=218633036018.221954264375"><img alt="Add to Slack" height="40" width="139" src="https://platform.slack-edge.com/img/add_to_slack.png" srcset="https://platform.slack-edge.com/img/add_to_slack.png 1x, https://platform.slack-edge.com/img/add_to_slack@2x.png 2x" /></a>
```

Comment:
client_id is not sensitive.  Appears to be something slack expects to be thrown around often.  client_secret is the only sensitive one, I believe.